middlewares/apiKey.js

```diff
@@ -0,0 +1,13 @@
+const validApiKey = "your_secret_api_key"; // Replace this with your actual API key
```

Comment:
do not expose API keys in the code, they should be store in an environment variable